waterloop-site/src/theme/styles.scss

```diff
@@ -1208,3 +1209,75 @@ textarea {
   flex-direction: column;
   justify-content: center;
 }
+
+.Block-Blog {
+  h2 a {
+    font-size: 18px;
+    font-weight: normal;
+    margin-left: 2rem;
+    cursor: pointer;
+  }
+}
+
+.Block-BlogPosts {
+  .PostsBlock-Blog {
+    width: 100%;
+    // display: inline-block;
+    .PostBlock-Blog {
+      display: inline-grid;
+      margin: 20px 0px;
+      transition: transform 250ms linear;
+      cursor: pointer;
+      .ContentBlock-Blog{
+        max-width: 300px;
+        margin: 0 auto;
+        border-radius: 10pt;
+        box-shadow: 0px 0px 8px 0px #ada8a8;
+        img {
+          height: 250px;
+          width: 100%;
+          border-top-left-radius: 10px;
+          border-top-right-radius: 10px;
+        }
+        .PostContent-Blog {
+          padding: 10px;
+          height: 200px;
+          .PostTitle-Blog {
+            font-size: 18px;
+            font-weight: bold;
+            font-style: italic;
+            margin-bottom: 5px;
+          }
+          .PostText-Blog {
+            font-size: 14px;
+        }
+
+        }
+      }
+    }
+    .PostBlock-Blog:hover {
+      transform: translateY(-10px);
+    }
+  }
+  @media (min-width: $tablet + 1) {
+      .PostBlock-Blog {
+        width: 33%;
+        .ContentBlock-Blog {
+          width: 95%;
+        }
+      }
+  }
+  @media (min-width: $mobile + 1) and (max-width: $tablet) {
+    .PostBlock-Blog {
+      width: 50%;
+      .ContentBlock-Blog {
+        width: 95%;
+      }
+    }
+}
+@media (max-width: $mobile) {
+    .PostBlock-Blog {
+      width: 100%;
+    }
+}
+}
```

Comment:
We are trying to go towards styled components rather than scss, use styled components next time.